app/org/sagebionetworks/bridge/models/studies/Subpopulation.java

```diff
@@ -0,0 +1,43 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Set;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoSubpopulation;
+import org.sagebionetworks.bridge.models.Criteria;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(as=DynamoSubpopulation.class)
+public interface Subpopulation extends Criteria {
+
+    public static Subpopulation create() {
+        return new DynamoSubpopulation();
+    }
+    
+    public void setStudyIdentifier(String studyIdentifier);
+    public String getStudyIdentifier();
+    
+    public void setGuid(String guid);
+    public String getGuid();
+    
+    public void setName(String name);
+    public String getName();
+    
+    public void setDescription(String description);
+    public String getDescription();
+    
+    public void setRequired(boolean required);
+    public boolean isRequired();
```

Comment:
What are the semantics of "required"? It looks like this is set only for the default subpop? iirc, we wanted to make the default subpop immutable, not just undeletable, so we can avoid the problem of no matching subpops.

We should comment this, since the semantics are unclear from the name.